sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -288,7 +288,7 @@ static const char *getFormatStr(RT::PiDeviceBinaryType Format) {
 }
 
 RT::PiProgram ProgramManager::createPIProgram(const RTDeviceBinaryImage &Img,
-                                              const context &Context) {
+                                              const context &Context) {                                          
```

Comment:
unneeded change

Reply:
I don't know why it happened but I already removed it in the commit above